pkg/network/ovn_kubernetes.go

```diff
@@ -39,6 +39,7 @@ const OVN_NB_PORT = "9641"
 const OVN_SB_PORT = "9642"
 const OVN_NB_RAFT_PORT = "9643"
 const OVN_SB_RAFT_PORT = "9644"
+const OVN_SB_ROUTE_PORT = "443"
```

Comment:
Ultimately we will need to get this from Status.

In fact, I suspect we're going to need to add a new response code to Bootstrap that says "you can continue, but not everything is deployed yet, please mark as Progressing".

Reply:
Added TODO comment.